src/parser/csql_grammar.y

```diff
@@ -3156,6 +3156,14 @@ create_stmt
 			      {
 				PT_ERROR (this_parser, node, "PUBLIC SYNONYM is not supported.");
 			      }
+
+			    if (node->info.synonym.target_name)
+			      {
+			        if (strchr (node->info.synonym.target_name->info.name.original, '@'))
+			          {
+				    node->info.synonym.dblinked = 1;
```

Comment:
The use of '@' in PT_NAME is a special case.
How about setting a flag for PT_NAME as well?

Reply:
dblinked flag is set at only this point. The flag would be not necessary.

Reply:
Thank you.